### PR TITLE
CSV download

### PR DESCRIPTION
**Pull request by baarkerlounger**
Currently this download all logs a user has permission to see (since we don't have filters yet). i.e. the Download does not consider pagination.

The CSV includes answer value labels rather than values (i.e. "Yes" rather than 1)

Link:
image

### CODE REVIEW

File: app/views/case_logs/_log_list.html.erb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <figure class="app-figure">
   <figcaption id="<%= title.dasherize %>" class="app-figure__caption">
-    <strong><%= pagy.count %></strong> total <%= title.downcase %>
+    <span class="govuk-!-margin-right-4">
+      <strong><%= pagy.count %></strong> total <%= title.downcase %>
+    </span>
+    <a class="govuk-link" download href="/logs.csv">Download (CSV)</a>
```
Comment (paulrobertlloyd) [inline app/views/case_logs/_log_list.html.erb]: Missed this in the prototype, but if I were to be super pedantic…

```suggestion
    <a class="govuk-link" download href="/logs.csv" type="text/csv">Download (CSV)</a>
```

**Review by paulrobertlloyd — APPROVED** [review]
I’m not sure the downloaded CSV should respect pagination (only search/filtering).

File: app/views/case_logs/_log_list.html.erb (hunk 1 above)
Comment (paulrobertlloyd) [inline app/views/case_logs/_log_list.html.erb]: We can also use the `download` attribute to give the downloaded CSV a filename… not sure what a useful name would be, though.

**Comment by baarkerlounger** [comment]
@paulrobertlloyd that was my assumption as well, was just being explicit about it in the description

File: app/views/case_logs/_log_list.html.erb (hunk 1 above)
Comment (baarkerlounger) [reply app/views/case_logs/_log_list.html.erb]: Filename is set in the controller currently

Comment (paulrobertlloyd) [reply app/views/case_logs/_log_list.html.erb]: That’s probably best; `download` isn’t supported by IE11

**Pull request merged**